conans/client/tools/win.py

```diff
@@ -93,8 +93,31 @@ def latest_vs_version_installed(output):
     return latest_visual_studio_version_installed(output=output)
 
 
+MSVS_DEFAULT_TOOLSETS = {"16": "v141",
+                         "15": "v141",
+                         "14": "v140",
+                         "12": "v120",
+                         "11": "v110",
+                         "10": "v100",
+                         "9": "v90",
+                         "8": "v80"}
+
+MSVS_VERSIONS = sorted(list(MSVS_DEFAULT_TOOLSETS.keys()), key=int)
+
+
+def msvs_toolset(settings):
+    toolset = settings.get_safe("compiler.toolset")
+    if not toolset:
+        vs_version = settings.get_safe("compiler.version")
+        try:
```

Comment:
this try/except could be simplified with ``toolset = MSVS_DEFAULT_TOOLSETS.get(vs_version)``

Reply:
simplified